automation/Jenkinsfile

```diff
@@ -375,7 +379,37 @@ pipeline {
         """        
       }
     }
-    
+    /*
+    *  Execute remove-organization.yamll which will remove an organisation from the network
+    */
+    stage('Remove an organisation') {
+      when {
+        not {
+          anyOf {
+            branch 'master';
```

Comment:
instead do when fabric?

Reply:
yep will do, was thinking of gradually removing the not branches when we do for other DLTs , but yeah for now just Fabric makes more sense